packages/core/src/components/card-container/card-container.tsx

```diff
@@ -0,0 +1,29 @@
+import { CardContainerMode } from "@dso-toolkit/sources";
```

Comment:
Ben ik niet gewend, normaal zouden we `CardContainerMode` in een file naast het component definieren. Even dubbelcheck bij oa. Accordion.

Reply:
Je hebt gelijk, in het bestand een rood kringeltje, maar verder geen error bij de build o.i.d.